### PR TITLE
Added Slack to the list of supported service

### PR DESCRIPTION
**Pull request by Croos3r**

### CODE REVIEW

File: backend/back/src/workflows/workflows.service.ts [hunk 1]
```diff
@@ -261,13 +260,6 @@ export class WorkflowsService {
 
 			if (reactions) {
 				this.logger.log(`Updating ${reactions.length} workflow ${workflowId} reactions...`);
-				this.logger.debug(
-					`Deleted workflow reactions: ${JSON.stringify(
-						await queryRunner.manager.delete(WorkflowArea, {
-							id: In(workflow.reactions.map((reaction) => reaction.id)),
-						}),
-					)}`,
-				);
```
Comment (RezaRahemtola) [inline backend/back/src/workflows/workflows.service.ts:270]: why did you remove this ? the reactions are not deleted now

Comment (Croos3r) [reply backend/back/src/workflows/workflows.service.ts:270]: Didn't really saw that ahaha, just tought of deleting the debug log

**Review by RezaRahemtola — APPROVED** [review]

**Review by EdenComp — APPROVED** [review]
